packages/@aws-cdk/aws-docdb/lib/cluster.ts

```diff
@@ -293,6 +293,11 @@ export class DatabaseCluster extends DatabaseClusterBase {
    */
   public readonly secret?: secretsmanager.ISecret;
 
+  /**
+   * The secret attached to this cluster
+   */
+  public readonly databaseSecret?: DatabaseSecret | undefined;
```

Comment:
the secret itself was not enough. We need the DatabaseSecret, because only it has the excludeCharacters property. But I think, we can change the type of the secret instead.

Reply:
> Also, we're missing a new unit test for the non-default case.

Do we? There is no logic handling a custom excludeCharacters string so I thought testing / ensuring that excludeCharacters inside the secret and the secretrotation match would be enough. Just my thoughts